third_party/intel/lib/Dialect/TritonGEN/IR/TritonGENDialect.cpp

```diff
@@ -40,9 +40,17 @@ void TritonGENDialect::initialize() {
 }
 
 int triton::TritonGEN::getSubgroupSize(Operation *op) {
-  spirv::TargetEnvAttr attr = spirv::lookupTargetEnv(op);
-  assert(attr && "Expecting valid target env attribute");
-  return attr.getResourceLimits().getSubgroupSize();
+  spirv::TargetEnvAttr spvEnvattr = spirv::lookupTargetEnv(op);
+  if (spvEnvattr)
+    return spvEnvattr.getResourceLimits().getSubgroupSize();
+  // FIXME: Remove the fallback code after llvm version updated to PR
+  // llvm/llvm-project#90972
```

Comment:
llvm/llvm-project#90972 also uses `spirv.target_env`. In order to remove the fallback code, `spirv.target_env` needs to be added in Triton.
```suggestion
  // FIXME: Remove the fallback code when `spirv.target_env` is added in Triton pipeline.
```